projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py

```diff
@@ -44,23 +44,25 @@ class JobConfig:
 
     def __init__(self, data_job_path: Union[pathlib.Path, str]):
         self._config_ini = configparser.ConfigParser()
-        self._config_file = os.path.join(data_job_path, "config.ini")
-
-        if not os.path.isfile(self._config_file):
-            raise VdkConfigurationError(
-                ErrorMessage(
-                    summary="Error while loading config.ini file",
-                    what="Cannot extract job Configuration",
-                    why=f"Configuration file config.ini is missing in data job path: {data_job_path}",
-                    consequences="Cannot deploy and configure the data job without config.ini file.",
-                    countermeasures="config.ini must be in the root of the data job folder. "
-                    "Make sure the file is created "
-                    "or double check the data job path is passed correctly.",
-                )
-            )
-        self._read_config_ini_file(
-            config_parser=self._config_ini, configuration_file_path=self._config_file
-        )
+        self._config_ini.read(str(data_job_path.joinpath("config.ini")))
```

Comment:
Hi, @tozka 
In the old times, this was the way we were doing it, https://github.com/vmware/versatile-data-kit/commit/06eb4a51a1eacc5f633d8b6ebdcef011fb682512

Reply:
Also, according to the documentation, it just returns an empty dataset, https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read

Reply:
Oh, ok.

 There's no testing done in the section. Before merging make sure to test it. As trivial as it may seem the change, it's aways possible to miss something. We may even add an automated test for that as we do not want to fail if config.ini is missing. 

It's approved so merge it when you feel it's ready.

Reply:
Agree, I tested it locally, so it seems to be working as expected.